CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ x.x.x Release notes (yyyy-MM-dd)
 ### Bugfixes
 
 * Use of KVC collection operators on Swift collection types no longer throws an exception.
+* The contents of `List` and `Optional` properties are now correctly preserved when copying
```

Comment:
That content also wasn't being preserved when initializing an object with the `Object(value:)` initializer IIRC.